src/sessions/__tests__/PageManager.test.ts

```diff
@@ -11,6 +14,48 @@ Object.defineProperty(document, 'referrer', {
 });
 Object.defineProperty(document, 'title', { value: 'Amazon AWS Console' });
 
+const mockFetch = jest.fn(
```

Comment:
Could add this to test-util and re-use by import in fetchPlugin and pageManager test files.